### DISCUSSION TITLE
What's this horizontal line for?

### DISCUSSION THREAD

**Discussion by madprops** [Taiko2k/Tauon#1192] [General]
![line234](https://github.com/user-attachments/assets/d3b70aec-fcbd-4dfc-9c71-7fe56bf0dc53)

Not sure why it appears there. I thought maybe it was because I skipped the song in the middle of it but doesn't seem to happen again when I do that.

**Comment by madprops** [comment]
I think I know what it is, it's a marker that appears when a certain % of the song was played, is my guess, as a way to know it was played.

**Reply by madprops** [reply]
Do they ever reset or does it stick after you played a song once?

**Reply by Taiko2k** [reply]
It stays and grows longer as you play the track more. Its indeed designed to show which tracks you have played and also show which you liked the most. You can disable this feature under MENU > Settings > Tracklist. There is also a 'star' shaped variant of this feature.
